src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.NotificationHubs;microsoft.notificationhubs/namespaces;Notification Hubs;Mobile;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.nutanix;microsoft.nutanix/nodes;Azure VMWare Solution;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.OpenEnergyPlatform;microsoft.openenergyplatform/energyservices;Azure Data manager for Energy;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/azurefirewall;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/clusters;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/networksecuritygroups;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/p2svpngateways;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.OperationalInsights;microsoft.operationalinsights/virtualnetworkgateways;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.operationalinsights;microsoft.operationalinsights/workspaces;Azure Monitor;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Orbital;microsoft.orbital/spacecrafts;Azure Orbital;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+MICROSOFT.ORBITAL;microsoft.orbital/terminals;Azure Orbital;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Peering;microsoft.peering/peerings;Azure Peering Service;Networking;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.PowerBIDedicated;microsoft.powerbidedicated/autoscalevcores;Power BI Embedded;Analytics;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.PowerBIDedicated;microsoft.powerbidedicated/capacities;Power BI Embedded;Analytics;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.PowerPlatform;microsoft.powerplatform/accounts;Power Platform;Integration;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Purview;microsoft.purview/accounts;Microsoft Purview;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Quantum;microsoft.quantum/workspaces;Azure Quantum;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.RecoveryServices;microsoft.recoveryservices/vaults;Azure Site Recovery;Storage;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Relay;microsoft.relay/namespaces;Azure Relay;Integration;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.SaaS;microsoft.saas/resources;Azure Marketplace;Other;Microsoft;Marketplace;Cloud;SaaS
+Microsoft.SaaS;microsoft.saas/saasresources;Azure Marketplace;Other;Microsoft;Marketplace;Cloud;SaaS
+Microsoft.Search;microsoft.search/searchservices;Azure Cognitive Search;Web;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Security;microsoft.security/mdeservers;Microsoft Defender for Endpoint;Multicloud;Microsoft;Cloud Provider;Cloud;PaaS?

```

Comment:
I agree with PaaS. Can you remove the `?`?

Reply:
Done